.github/workflows/build_prod.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - run: rm dist/.nojekyll
       - run: mv dist/robots.prod.txt dist/robots.txt
       - name: Upload artifact
-        uses: actions/upload-artifact@v1
```

Comment:
You can even use v4. I don't think the breaking changes would have any impact on this repository. (Same for download artifact)